Makefile

```diff
@@ -159,21 +159,21 @@ e2e-tests: test-e2e-setup
 
 test-e2e-setup:
 	@echo "Seting up E2E Tests environment..."
-ifdef COMPONENT_IMAGE_NAME
+ifdef COMPONENT_IMAGE_PIPELINE
 	# override the image for the e2e test
-	@./cicd-scripts/setup-e2e-tests.sh -a install -i $(COMPONENT_IMAGE_NAME)
+	@./cicd-scripts/setup-e2e-tests.sh -a install -p $(COMPONENT_IMAGE_PIPELINE)
+endif
+ifdef COMPONENT_IMAGE_NAMES
+	# override the image for the e2e test
```

Comment:
do we still need this? `COMPONENT_IMAGE_PIPELINE` can be used for local test?

Reply:
forget that, I got that, the pipeline images have different format.

Reply:
`COMPONENT_IMAGE_PIPELINE` is something like `registry.build01.ci.openshift.org/ci-op-1ts27vc8/pipeline` so just use component name as image tag. that is why leave `COMPONENT_IMAGE_NAMES` for local test.